manifests/plugin/gutterball/config.pp

```diff
@@ -0,0 +1,15 @@
+# gutterball config
+class katello::plugin::gutterball::config(
+    $foreman_user  = 'foreman',
+    $foreman_group = 'foreman',
+    $foreman_plugins_dir = '/etc/foreman/plugins',
+    $url = "https://${fqdn}:8443/gutterball",
+  ){
+
+  file { "${foreman_plugins_dir}/foreman_gutterball.yaml":
```

Comment:
I prefer things are explicit and match the plugin name. That way there is
no translation or discrepancy whenever you are looking for something
related. I think that the namespace of the plugins config and the file
containing it should remain the same.
On Feb 3, 2015 7:58 AM, "Ewoud Kohl van Wijngaarden" <
notifications@github.com> wrote:

> In manifests/plugin/gutterball/config.pp
> <https://github.com/Katello/puppet-katello/pull/58#discussion_r24001396>:
>
> > @@ -0,0 +1,15 @@
> > +# gutterball config
> > +class katello::plugin::gutterball::config(
> > +    $foreman_user  = 'foreman',
> > +    $foreman_group = 'foreman',
> > +    $foreman_plugins_dir = '/etc/foreman/plugins',
> > +    $url = "https://${fqdn}:8443/gutterball",
> > +  ){
> > +
> > +  file { "${foreman_plugins_dir}/foreman_gutterball.yaml":
>
> I'd name this simply gutterball.yaml since foreman is implied.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Katello/puppet-katello/pull/58/files#r24001396>.
>